src/main/java/com/googlecode/lanterna/gui2/AbstractWindow.java

```diff
@@ -149,14 +149,30 @@ public TerminalPosition toGlobal(TerminalPosition localPosition) {
     }
 
     @Override
+    @Deprecated
     public TerminalPosition fromGlobal(TerminalPosition globalPosition) {
+        return fromContentGlobal(globalPosition);
+    }
+    
+    @Override
+    public TerminalPosition fromContentGlobal(TerminalPosition globalPosition) {
         if(globalPosition == null || lastKnownPosition == null) {
             return null;
         }
         return globalPosition.withRelative(
                 -lastKnownPosition.getColumn() - contentOffset.getColumn(),
                 -lastKnownPosition.getRow() - contentOffset.getRow());
     }
+    
+    @Override
+    public TerminalPosition fromDecoratedGlobal(TerminalPosition globalPosition) {
```

Comment:
Are you calling this method anywhere? I don't see any usages of it

Reply:
Yep, MultiWindowTextGUI for determination of which window is clicked on.